x/orderbook/handler.go

```diff
@@ -0,0 +1,100 @@
+package orderbook
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+	"github.com/iov-one/weave/x"
+)
+
+const (
+	packageName            = "orderbook"
+	newOrderBookCost int64 = 100
+)
+
+// RegisterQuery registers exchange buckets for querying.
+func RegisterQuery(qr weave.QueryRouter) {
+	NewMarketBucket().Register("markets", qr)
+	NewOrderBookBucket().Register("orderbooks", qr)
+}
+
+// RegisterRoutes registers handlers for orderbook message processing.
+func RegisterRoutes(r weave.Registry, auth x.Authenticator, issuer weave.Address) {
+	r = migration.SchemaMigratingRegistry(packageName, r)
+
+	//	r.Handle(CreateOrderBookMsg.Path(), )
```

Comment:
Why don't you register here, even if handler not fully functional?
Also, maybe you need `CreateOrderBookMsg{}.Path()`